client/app/dedicatedUniverseComponents/price/index.js

```diff
@@ -0,0 +1,17 @@
+import angular from 'angular';
+import translate from 'angular-translate';
+import translateAsyncLoader from '@ovh-ux/translate-async-loader';
+
+import ducPriceFilter from './price.filter';
+
+const moduleName = 'ducPrice';
+
+angular
+  .module(moduleName, [
+    translate,
+    translateAsyncLoader,
+  ])
+  .filter('ducPrice', ducPriceFilter)
```

Comment:
I've just one question according to the `price` filter from [`ovh-utils-angular`](https://github.com/ovh-ux/ovh-utils-angular/blob/master/src/price/price-filter.js#L1).

How we are sure that the filter you've just updated is not the same as the `ovh-utils-angular` one?

Reply:
ping @cbourgois